handlers.js

```diff
@@ -1,8 +1,7 @@
-const jwt = require('jsonwebtoken')
-const { totp } = require('otplib')
-const { v4: uuidv4 } = require('uuid')
+import jwt from '@tsndr/cloudflare-worker-jwt'
```

Comment:
perhaps keep the import style consistent (others are `require` in this file)

Reply:
done